tools/clang/test/CodeGenSPIRV/fn.forward-declaration.hlsl

```diff
@@ -0,0 +1,16 @@
+// Run: %dxc -T ps_6_0 -E main
+
+float4 main() : SV_Target
+{
+  float MulBy2(float f);
+
+  // CHECK: OpFunctionCall %float %MulBy2 %param_var_f
+  return float4(MulBy2(.25), 1, 0, 1);
+}
```

Comment:
Does it handle the following case?
```
struct foo
{
  ...
  float4 memberFunction() {
    float MulBy2(float f);
    return float4(MulBy2(.25), 1, 0, 1);
  }
};
```

Reply:
Yes, it works. I've added this to the test.